airflow/www/static/js/dag/details/graph/utils.ts

```diff
@@ -109,16 +117,17 @@ export const flattenNodes = ({
 };
 
 export const nodeColor = ({
-  data: { height, width, instance },
+  data: { height, width, instance, childCount, isActive },
 }: ReactFlowNode<CustomNodeProps>) => {
+  let opacity = "90";
+  let color = "#cccccc";
   if (!height || !width) return "";
-  if (width > 200 || height > 60) {
-    return "#cccccc50";
-  }
-  if (instance?.state) {
-    return `${Color(stateColors[instance.state]).hex()}90`;
-  }
-  return "#cccccc90";
+  if (instance?.state && !childCount)
```

Comment:
Also, figured that checking the child count is better than checking the node size to determine if the node is a group in the minimap